MANIFEST.in

```diff
@@ -1,9 +1,13 @@
+graft crfsuite
+graft liblbfgs
+graft include
+graft pycrfsuite
+graft tests
+
+include LICENSE.txt
 include README.rst
 include CHANGES.rst
 include update_cpp.sh
-include include/inttypes.h
 
-recursive-include crfsuite *
-recursive-include liblbfgs *
-recursive-include tests *.py
-recursive-include pycrfsuite *.py *.pxd *.pyx *.cpp *.hpp
+global-exclude .py[cod] __pycache__
```

Comment:
It also requires automatic Travis releases; without them I'd prefer to keep exclude-by-default.